relay-cogs/src/lib.rs

```diff
@@ -0,0 +1,189 @@
+use std::time::Duration;
+
+mod cogs;
+mod recorder;
+#[cfg(test)]
+mod test;
+mod utils;
+
+pub use cogs::*;
+pub use recorder::*;
+#[cfg(test)]
+pub use test::*;
+
+/// Resource ID as tracked in COGS.
+///
+/// Infrastructure costs are labeled with a resource id,
+/// these costs need to be broken down further by the application
+/// by [app features](AppFeature).
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
+pub enum ResourceId {
```

Comment:
What other resource IDs may Relay have? If it's only going to be one, I'd hardcode it instead of having it as a variable.

Reply:
Once we want to start splitting costs like GCP network traffic, we'll need a second resource id. Also it does follow the [COGS Architecture](https://github.com/getsentry/etl/blob/master/workspace/dags/sql/cogs_v2/docs/architecture.md).